src/lib/tabs/tabs.ts

```diff
@@ -30,6 +37,7 @@ import {MdTabNavBar, MdTabLink} from './tab-nav-bar/tab-nav-bar';
 import {MdInkBar} from './ink-bar';
 import {Observable} from 'rxjs/Observable';
 import 'rxjs/add/operator/map';
+import {PortalHostDirective} from '../core/portal/portal-directives';
```

Comment:
It might make more sense to add this to the existing import list on line 19 from `../core`. I think it's exported there too.